lib/modules/watcher/index.js

```diff
@@ -0,0 +1,180 @@
+let chokidar = require('chokidar');
+let path = require('path');
+
+let fs = require('../../core/fs.js');
+
+// TODO: this should be receiving the config object not re-reading the
+// embark.json file
+class Watcher {
```

Comment:
I renamed this to `Watcher` instead of `Watch`. Thought this was a good opportunity to do this. Let me know if you disagree with this.